SwiftCheck/Test.swift

```diff
@@ -488,8 +488,8 @@ internal func printDistributionGraph(st : State) {
 		return (n < 10 ? " " : "") + "\(n)" + "% "
 	}
 
-	let gAllLabels = st.collected.map({ s in
-		return Array(s).filter({ t in st.labels[t] == .Some(0) }).reduce("", combine: { (l, r) in l + ", " + r })
+	let gAllLabels = st.collected.map({ (s: Set<String>) in
```

Comment:
It's no problem.  Our formatting rules are a bit weird, but this one comes from me being paranoid and using too many languages where `<ident>: <type>` isn't allowed.